src/adapters/sonic/paintswap/index.ts

```diff
@@ -0,0 +1,114 @@
+import { TChainName, TAddress } from 'src/types/chains.js';
+import { autoInjectable } from 'tsyringe';
+import { Logger } from 'tslog';
+import { getChainByName, objectToQueryString } from '../../../utils/index.ts';
+import type { INFTDataAdapter, IAdapter } from '../../../types/adapter.d.ts';
```

Comment:
```suggestion
import type { INFTDataAdapter } from '../../../types/adapter.d.ts';
```

And `INFTDataAdapter` is not used in this file

Reply:
INFTDataAdapter will be our unified adapter for NFT Balance data (same like token). 

After I migrate Reservoir service from OCB, we will decide the final format of a universal NFTDataAdapter (and unified type as well)

Reply:
Update: Implement the unified type (TNftBalance) with IOnchainNftAdapter already